server/app.js

```diff
@@ -17,6 +21,22 @@ app.use(compression({ threshold: 0 }));
 mongoose.Promise = bluebird;
 mongoose.connect(process.env.MONGO_URI);
 
+const logDirectory = path.join(__dirname, 'log');
```

Comment:
We use pm2 to actually run the application. It will manage starting, stoping, and restarting a process when it errors. pm2 will also run the application in a cluster. Meaning there will be four versions of the application running on the same server.

Because of this, pm2 intercepts anything printed to std:out by our nodejs app. It will manage the log file and things like log rotation. 

This means that all we need to do is add morgan and we can assume pm2 will take care of all the rest.

The other additions to this PR are not necessary.